lib/puppet_blacksmith/modulefile.rb

```diff
@@ -46,7 +46,7 @@ def bump!(level = :patch)
     end
 
     %i[major minor patch full].each do |level|
-      define_method("bump_#{level}!") { bump!(level) }
+      define_method(:"bump_#{level}!") { bump!(level) }
```

Comment:
I don't know if this is sane or not, but it was done by the rubocop safe autocorrect